### PR TITLE
Feature/add banner

### PR DESCRIPTION
**Pull request by mattbaumann1**
I was unable to create the failures in order to test some of the error pathways, but this should provide a simple admin banner.

### CODE REVIEW

File: spec/system/banners/banners_spec.rb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      context 'home page' do
+        context 'header links' do
+          scenario 'displays login link' do
+            expect(page).to have_link 'Admin'
+          end
+
+          scenario 'displays the Banners link for admins' do
+            click_link 'Admin'
+            click_link 'Banners'
+            expect(current_path).to eq("/banners")
```
Comment (jseraf) [inline spec/system/banners/banners_spec.rb]: This and other admin paths (All Users, All Grants) could be moved into an admin namespace. (admin/banners, admin/users). Will discuss and open a ticket about it.

File: app/policies/banner_policy.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+class BannerPolicy < ApplicationPolicy
+
+  class Scope < Scope
+    def resolve
+      if user.system_admin?
+        scope.all
+      end
+    end
+  end
```
Comment (jseraf) [inline app/policies/banner_policy.rb]: Can the scope be bypassed? Index is for admins only

File: app/views/banners/_form.html.haml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    = f.check_box :visible, value: banner.visible
+.grid-x.grid-padding-y
+  .cell
+    = f.label :body, 'Body'
+    = f.hidden_field :body, id: :banner_body
+    %trix-editor(input='banner_body')
+.grid-x.grid-padding-y
+  .cell.small-12.medium-6
+    = f.label :created_at, 'Created At'
+    = f.text_field :created_at, value: banner.created_at, disabled: true, class: 'small-12 medium-4'
```
Comment (jseraf) [inline app/views/banners/_form.html.haml]: Pretty sure value: is not needed here (or elsewhere)

File: spec/system/banners/banners_spec.rb (hunk 1 above)
Comment (mattbaumann1) [reply spec/system/banners/banners_spec.rb]: I like this idea, but it should be a new ticket and planned into a future sprint.

File: app/views/banners/_form.html.haml (hunk 3 above)
Comment (mattbaumann1) [reply app/views/banners/_form.html.haml]: Removed.

**Pull request merged**